### ISSUE TITLE
input[type="range"] pseudo selector for current value (akin to aria-valuenow)

### ISSUE THREAD

**Issue by michaelcpuckett** [w3c/csswg-drafts#6354]
I would like to be able to use a native `<input type="range">` (which comes with arrow keyboard support) and style it (or its siblings, etc) based on its current value.

If I create a comparable ARIA slider widget with JavaScript, I can style it based on the `[aria-valuenow]` attribute. I think the same feature should exist for the native element to enable more native interactivity for non-JS users.

**Comment by Loirooriol** [comment]
This would probably need numerical comparisons in order to really be useful. #5890 covers that for attribute selectors.

**Comment by yisibl** [comment]
So you want to get the value of the current input via CSS?

For example, `env(input-current-value)`

**Comment by michaelcpuckett** [comment]
No, I just want the current value rendered into the range input's HTML or available as a CSS pseudo selector on it so it can be styled based on the value.

**Comment by xiaochengh** [comment]
> So you want to get the value of the current input via CSS?
>
> For example, env(input-current-value)

That sounds like the advanced `attr()` function, though no browser supports it yet
